pkg/asset/installconfig/openstack/validation/platform.go

```diff
@@ -37,26 +40,79 @@ func ValidatePlatform(p *openstack.Platform, n *types.Networking, ci *CloudInfo)
 	return allErrs
 }
 
-// validateMachinesSubnet validates the machines subnet and enforces proper byo subnet usage and returns a list of all validation errors
-func validateMachinesSubnet(p *openstack.Platform, n *types.Networking, ci *CloudInfo, fldPath *field.Path) (allErrs field.ErrorList) {
```

Comment:
This validation function is being removed because the machineSubnets config would be converted to controlPlanePort's network config before it is validated (i.e when the deprecated config is provided by the user and not the new config)?

Reply:
Yes.
The conversion happens when [loading the install-config](https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/installconfig.go#L118), which happens before [the platform validation.](https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/installconfig.go#L175)